src/Schema/Keywords/Type.php

```diff
@@ -48,8 +49,7 @@ public function validate($data, string $type, ?string $format = null) : void
                     throw TypeMismatch::becauseTypeDoesNotMatch(CebeType::OBJECT, $data);
                 }
                 break;
-            case 'array':
-                // no constant here yet https://github.com/cebe/php-openapi/pull/24
+            case CebeType::ARRAY:
```

Comment:
composer.json should require at least `"cebe/php-openapi": "^1.0.1"` as constant did not exist before that version.

Reply:
there should be no reason to use anything below 1.2 though, so if you bump the constraint you can go with `"^1.2"` directly.